docs/core/additional-tools/self-signed-certificates-guide.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Generate Self-Signed Certificates Overview
+description: An overview of the Microsoft dotnet dev-certs tool that adds functionality for .NET Core and ASP.NET Core projects, and other options for using self-signed certificates.
+author: angee
+ms.date: 11/19/2020
+---
+
+# Generating Self-Signed Certificates
+
+When using self-signed certificates, there's different ways to create and use them for development and testing scenarios.  In this guide, you'll cover using self-signed certificates with `dotnet dev-certs`, and other options like `PowerShell` and `OpenSSL`.
+
+You can then validate that the certificate will load using an example such as an [aspnetcore app](https://github.com/dotnet/dotnet-docker/blob/master/samples/run-aspnetcore-https-development.md) hosted in a container.
+
+## Prerequisites
+
+In the sample, you can utilize either `.netcore 3.1` or `.net 5`.
+
+For `dotnet dev-certs`, be sure to have the appropriate version of `dotnet` installed:
+
+* [Install dotnet on Windows](../install/windows.md)
+* [Install dotnet on Linux](../install/linux.md)
+* [Install dotnet on macOS](../install/macos.md)
+
+This sample requires [Docker 17.06](https://docs.docker.com/release-notes/docker-ce) or later of the [Docker client](https://www.docker.com/products/docker).
+
+## Prepare Sample App
```

Comment:
I didn't mark this on every heading, but all headings should be sentence case, not title case:

```suggestion
## Prepare sample app
```

Reply:
Updated, thank you!